src/app/public/plugin-resources/code-examples/checkbox/checkbox-demo.component.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'sky-checkbox-demo',
```

Comment:
Shouldn't the selector start with `app` instead of `sky`?

Reply:
Done.